gremlin-dotnet/src/Gremlin.Net/Process/Traversal/Strategy/Decoration/SubgraphStrategy.cs

```diff
@@ -53,6 +54,10 @@ public SubgraphStrategy() : base(JavaFqcn)
                 Configuration["edges"] = edges;
             if (vertexProperties != null)
                 Configuration["vertexProperties"] = vertexProperties;
+
+            // hate to write this by default so only going to write it if it's false which is the non-default value
+            if (!checkAdjacentVertices)
+                Configuration["checkAdjacentVertices"] = checkAdjacentVertices;
```

Comment:
Really just a nit, but my IDE (Rider) is complaining that `checkAdjacentVertices` will always be `false` here and suggests to change this to `Configuration["checkAdjacentVertices"] = false;` to make this more obvious.